File: my_message/src/talker.cpp
#include <sstream>
#include "ros/ros.h"
#include "std_msgs/String.h"

int main(int argc, char **argv)
{
    ros::init(argc, argv, "talker");
    ros::NodeHandle nh;
    ros::Publisher pub = nh.advertise<std_msgs::String>("chatter", 10);
    ros::Rate loop_rate(2);

    int count = 0;

    while(ros::ok())
    {
        std_msgs::String msg;
        std::stringstream ss;
        
        ss << "hello world " << count;
        msg.data = ss.str();

        ROS_INFO("%s", msg.data.c_str());
        pub.publish(msg);

        count ++;
        
        ros::spinOnce();
        loop_rate.sleep();
    }

    return 0;
}

File: my_message/scripts/talker.py
#!/usr/bin/env python

# -*- coding: utf-8 -*-
# @Author: IBNBlank
# @Date:   2019-01-28 22:50:17
# @Last Modified by:   IBNBlank
# @Last Modified time: 2019-01-29 21:54:21

import rospy
from std_msgs.msg import String



if __name__ == '__main__':
    rospy.init_node("talker", anonymous=True)
    pub = rospy.Publisher("chatter", String, queue_size=10)
    rate = rospy.Rate(2)

    count = 0

    while not rospy.is_shutdown():
        msg_str = "hello world {}".format(count)

        rospy.loginfo(msg_str)
        pub.publish(msg_str)

        count += 1

        rate.sleep()
